src/main/java/mate/academy/bookapp/model/Book.java

```diff
@@ -0,0 +1,26 @@
+package mate.academy.bookapp.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data

```

Comment:
Trying to avoid using `@Data` for entities because it could provide some problems. I suggest you to read an [article](https://medium.com/@miguelangelperezdiaz444/the-hidden-dangers-of-lombok-annotations-in-your-java-code-what-you-need-to-know-8acdce2d6b89) about it